slurm/install/installlib.py

```diff
@@ -427,3 +493,183 @@ def set_hostname(
             "import jetpack.converge as jc; jc._send_installation_status('warning')",
         ],
     )
+
+
+class CCNode:
+    """
+    Simple CycleCloud node representation.
```

Comment:
again maybe a better fit for scalelib? in fact scalelib should already have a node representation.